src/util/helpers.js

```diff
@@ -0,0 +1,34 @@
+export function sortByBalance(wallets) {
+  return wallets
+    .filter(wallet => parseFloat(wallet.balance) !== 0)
+    .sort((a, b) => b.balance - a.balance);
+}
+
+// export function combineWallets(addresses, walletContracts) {
+export function combineWallets(addr, wc) {
+  /* scrub for undefined parameters */
+  let addresses = addr;
+  let walletContracts = wc;
+  if (!addr) addresses = {}
```

Comment:
nit: i don't love one-liner ifs, i always prefer to have curly braces around, @marcgarreau what do you think

Reply:
I like one-liners and usually reserve them for 'gatekeeping' at the top of functions - just a style I prefer and have picked up along the way. I've removed them from this PR.